isso/ext/notifications.py

```diff
@@ -168,8 +194,8 @@ def notify_users(self, thread, comment):
                 email = comment_to_notify["email"]
                 if "email" in comment_to_notify and comment_to_notify["notification"] and email not in notified \
                         and comment_to_notify["id"] != comment["id"] and email != comment["email"]:
-                    body = self.format(thread, comment, parent_comment, email, admin=False)
-                    subject = "Re: New comment posted on %s" % thread["title"]
+                    body = self.format(thread, comment, parent_comment, comment_to_notify)
```

Comment:
comment_to_notify is passed in to the "recipient" argument. That seems wrong?

Reply:
That's not wrong, the whole comment record of the recipient is passed to the parameter recipient. Then things like recipient["id"] in the record can be used, not just recipient["email"]